aspnet/fundamentals/static-files.rst

```diff
@@ -280,21 +280,9 @@ Now, if the user attempts to browse to any file with an extension of ``.myapp``,
 IIS Considerations
 ------------------
 
-IIS (Internet Information Server) has a native static file module that is independent of the ASP.NET static file middleware components that you've learned about in this article. As the ASP.NET modules are run before the IIS native module, they take precedence over the IIS native module. As of `ASP.NET Beta 7 <https://github.com/aspnet/Announcements#54>`_, the IIS host has changed so that requests that are not handled by ASP.NET will return empty 404 responses instead of allowing the IIS native modules to run. To opt into running the IIS native modules, add the following call to the end of ``Startup.Configure``.
+IIS (Internet Information Server) acts as a reverse proxy which means that IIS HTTP Platform Handler doesn’t treat requests over static files any different from requests over dynamic content. It implies that the ASP.NET Core application should serves its static content (through the static file middleware) because the IIS native static file module won’t do it.
```

Comment:
How about this?: "ASP.NET 5 applications hosted in IIS use the HTTP platform handler to forward all requests to the application including requests for static files. The IIS static file handler is not used because it won’t get a chance to handle the request before it is handled by the HTTP."